lms/migrations/versions/b99c4d910801_disable_provisioning_for_non-whitelisted_.py

```diff
@@ -0,0 +1,96 @@
+"""
+Disabled provisioning for non-whitelisted consumer keys.
```

Comment:
Presumably this migration is intended as a one-time-use only. Would it be possible to expand this comment a little to help future devs understand this in context?

Reply:
As with any migration it'll only ever be run once (well, once on QA, once and prod, and up to once once in each of our dev envs). You'd only need to use the downgrade and then possibly use the upgrade again if something had gone wrong. Each time you ran the upgrade it would do the same thing: set `provisioning` to `False` for all non-whitelisted application instances.